notebook/jupyter_notebook_config.py

```diff
@@ -11,6 +11,7 @@
 c.NotebookApp.ip = '0.0.0.0'
 c.NotebookApp.port = 8888
 c.NotebookApp.open_browser = False
+c.NotebookApp.tornado_settings = {"xsrf_cookies": False}
```

Comment:
how does that set the cookie?

Reply:
It just sets the config to skip validating the XSRF cookies. We pass this config option from the notebook service app each time, I believe it makes more sense to just disable it here.